packages/frontend/src/components/wallet-migration/metrics.js

```diff
@@ -88,3 +93,16 @@ export const resetUserState = () => {
     }
     return rudderanalytics.reset();
 };
+
+export function accountIdToHash(accountId) {
+    if (!accountIdHash)  {
```

Comment:
Introducing state here now has introduced the potential for an event to be reported under the wrong accountId. Can we please compute the hash every time so as not to add any further complexity to event reporting?